test/auth.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Unit tests for the logon widget's 'Auth' service.
+ *
+ * @todo  Move login tests to this file.
+ */
+'use strict';
+
+var Auth = require('../scripts/lib/auth');
+var Cookies = require('js-cookie');
```

Comment:
js-cookie uses [`Cookies` in their docs](https://github.com/js-cookie/js-cookie#basic-usage). Not sure why I capitalized `Auth`. I can change it.